third_party/WebKit/Source/modules/webcl/WebCLKernelArgInfoProvider.cpp

```diff
@@ -157,8 +157,8 @@ void WebCLKernelArgInfoProvider::parseAndAppendDeclaration(const String& argumen
 
 String WebCLKernelArgInfoProvider::extractAddressQualifier(Vector<String>& declarationStrVector)
 {
-    static AtomicString* privateUnderlined = new AtomicString("__private", AtomicString::ConstructFromLiteral);
-    static AtomicString* private = new AtomicString("private", AtomicString::ConstructFromLiteral);
+    static AtomicString* privateQualifierUnderlined = new AtomicString("__privateQualifier", AtomicString::ConstructFromLiteral);
+    static AtomicString* privateQualifier = new AtomicString("privateQualifier", AtomicString::ConstructFromLiteral);
```

Comment:
Please try to name all qualifiers (private / global / read_only / write_only / ...) in a consistent way.